source/_integrations/melcloud.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: MELCloud
+description: MELCloud integration
+logo: home-assistant.png
+ha_category:
+  - Climate
+ha_release: 0.106
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@vilppuvuorinen'
+---
+
+The `melcloud` integration integrates Mitsubishi Electric's MELCloud enabled devices into Home Assistant.
+
+## Device support
+
+- Air-to-Air heat pumps, i.e. AC units - **Supported**
+- Air-to-Water heat pumps - **Work in progress**
+- Energy recovery ventilators - **Not supported**
+- Other - **Not supported**
+
+## Configuration
+
+MELCloud integration should be configured through the Home Assistant user interface using MELCloud  login details. This is not optimal, but **the provided password is not stored**.
+
+The stored access token can later be updated by adding the MELCloud integration again with the same email address.
+
+Configuration is also possible through `configuration.yaml`. The token can be found from `X-MitsContextKey` header when logged into MELCloud with 'English' and 'Remember me'.
+
+```yaml
+melcloud:
+  token: xxxxxxxxxxxxxxxxxxx
+```
+
+{% configuration %}
+token:
+  description: X-MitsContextKey access token
+  required: true
+  type: string
+{% endconfiguration %}
+
+## Air-to-Air device
+
+Air-to-Air heat pump provides `climate` and `sensor` platforms.
+
+### Climate
+
+Following parameters can be controlled throught the `climate` platform:
+
+- Target temperature
+- Operation mode (HVAC mode) with emulated power off.
```

Comment:
Is "emulated power off" understandable for end users?

Reply:
You'll never know. I tried to clarify this a bit.